lib/sqlbuilder/builder_test.go

```diff
@@ -349,11 +349,16 @@ func TestSelect(t *testing.T) {
 	)
 
 	assert.Equal(
-		`SELECT * FROM "artist" LIMIT -1 OFFSET 5`,
```

Comment:
That sounds good:

```
q.Where(db.Reset).Limit(db.Reset)
```

i like how it sounds, but let's implement that idea later on a different issue.